test_util/launch.py

```diff
@@ -1,20 +1,169 @@
-"""
+"""DC/OS Launch
 
 Usage:
-dcos-launch create [--wait] [--dump-info=cluster_info.json] <config=config.yaml>
-dcos-launch wait <path=cluster_info.json>
-dcos-launch describe <path=cluster_info.json>
-dcos-launch delete <path=cluster_info.json>
+  dcos-launch create [--config-path=<path>] [--info-path=<path>]
```

Comment:
Adding 'output' does not help. The info file that is produced by create is consumed by wait, delete, and describe. So, the user would use one special arg to point to where they want to the info json to appear and then a totally different arg to point to where they want that same thing to be found.

Furthermore, I believe this is well covered by the documentation that is printed out when you do --help:
```
  --info-path=<path>    JSON file output by create and consumed by wait, describe,
                        and delete [default: cluster_info.json].
```